### PR TITLE
update shell_plus for jupyterlab 3

### PR DESCRIPTION
**Pull request by terencehonles**

### CODE REVIEW

**Comment by trbs** [comment]
(build failure is unrelated)

File: django_extensions/management/commands/shell_plus.py [hunk 1]
```diff
@@ -203,7 +204,7 @@ def generate_kernel_specs(self, app, ipython_arguments):
 
         return {'django_extensions': ks}
 
-    def run_notebookapp(self, app, options, use_kernel_specs=True):
+    def run_notebookapp(self, initialize, options, use_kernel_specs=True):
```
Comment (trbs) [inline django_extensions/management/commands/shell_plus.py:207]: Can be use something else then `initialize` here ? maybe `app_init` or something. Which would more clearly describe it's purpose...

Comment (terencehonles) [reply django_extensions/management/commands/shell_plus.py]: done

**Comment by terencehonles** [comment]
@trbs is there anything else that should be adjusted before this PR is merged?

**Comment by trbs** [comment]
Thanks !

**Pull request merged**